great_expectations/expectations/metrics/column_aggregate_metrics/column_descriptive_stats.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, NamedTuple, Optional
+
+from great_expectations.compatibility.typing_extensions import override
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.metrics.column_aggregate_metric_provider import (
+    ColumnAggregateMetricProvider,
+)
+from great_expectations.expectations.metrics.metric_provider import metric_value
+from great_expectations.validator.metric_configuration import MetricConfiguration
+
+if TYPE_CHECKING:
+    from great_expectations.expectations.expectation_configuration import (
+        ExpectationConfiguration,
+    )
+
+
+class DescriptiveStats(NamedTuple):
```

Comment:
Not a blocker, but why a named tuple? I guess I would have expected a dataclass, since using this as a tuple seems like an easy way to mix up which value is which.

Reply:
I guess at first I was just going to return 4 values, which by default is a tuple, and then I named those values. I actually see now that the consumer uses a `BaseModel` so I think I'll just do that. It should functionally work the same regardless though...

Reply:
Yeah, I just feel like if it's used as a tuple, 4 values of the same type makes it a bit easier to e.g. confuse mean with standard dev.